test/150_connect_forget_test.sh

```diff
@@ -15,8 +15,8 @@ assert_peers() {
 
 start_suite "Connecting and forgetting routers after launch"
 
-weave_on $HOST1 launch
-weave_on $HOST2 launch
+weave_on $HOST1 launch-router
+weave_on $HOST2 launch-router
```

Comment:
> why the change?

Hah yes this one was a bit non-obvious - it's to do with IPAM. As far as I can tell using `weave launch` meant that two rings would form, as each peer allocated an IP for weaveDNS - this meant that the peers couldn't connect later due to an invalid state error. I'm guessing that happens in a lazy fashion, so if nothing asks for an IP then the peering is successful.

Let me try `weave launch -initpeercount` instead.

> Also note that it's using stop and launch below.

Good spot - will adjust.